packages/contracts-bedrock/src/L1/OPContractsManager.sol

```diff
@@ -30,176 +30,221 @@ import { IL1ERC721Bridge } from "interfaces/L1/IL1ERC721Bridge.sol";
 import { IL1StandardBridge } from "interfaces/L1/IL1StandardBridge.sol";
 import { IOptimismMintableERC20Factory } from "interfaces/universal/IOptimismMintableERC20Factory.sol";
 import { IHasSuperchainConfig } from "interfaces/L1/IHasSuperchainConfig.sol";
+import { ISystemConfigInterop } from "interfaces/L1/ISystemConfigInterop.sol";
 
-contract OPContractsManager is ISemver {
-    // -------- Structs --------
+contract LibOPContractsManager {
```

Comment:
I prefer not to use "Lib" unless it's actually a solidity `library`. "Base" or "Utils" or "Common" or something else would be fine.